sdks/python/apache_beam/typehints/batch.py

```diff
@@ -104,6 +110,12 @@ def __hash__(self) -> int:
 
 
 class ListBatchConverter(BatchConverter):
+  SAMPLE_FRACTION = 0.2
```

Comment:
Similarly here, this is just an arbitrary sample rate. Should we try harder to keep this consistent with the reservoir sampling done in `OperationCounters`?

Reply:
Let's add an upper bound here.

Reply:
Done.